properties.schema

```diff
@@ -48,6 +48,15 @@
       "validators": [],
       "help": "Allow the user to view the 'model answer' if they answer the question incorrectly?"
     },
+    "_canShowFeedback": {
+      "type":"boolean",
+      "required":true,
+      "default": true,
+      "title": "Display Feedback",
+      "inputType": "Checkbox",
+      "validators": [],
+      "help": "Select 'true' to allow the user to view feedback on their answer"
```

Comment:
could you leave the help out? I don't think it really adds anything; none of the other question components have help for this